fink_science/random_forest_snia/processor.py

```diff
@@ -276,9 +276,9 @@ def extract_features_rf_snia(jd, fid, magpsf, sigmapsf, cdsxmatch, ndethist) ->
 @pandas_udf(DoubleType(), PandasUDFType.SCALAR)
 def rfscore_sigmoid_elasticc(
         midPointTai, filterName, psFlux, psFluxErr,
-        cdsxmatch, nobs, ra, dec, hostgal_ra, hostgal_dec,
-        hostgal_zphot, hostgal_zphot_err,
-        mwebv, maxduration=None,
+        cdsxmatch, nobs, ra, dec, hostgal_ra, hostgal_dec, hostgal_snsep,
+        hostgal_zphot, hostgal_zphot_err, 
+        mwebv, mwebv_err, maxduration=None,
```

Comment:
Since you modified the constructor, you need to udpate the test (lines 339-345)

Reply:
I am not sure I know what I am doing, but I modified it